mysql-schema/032-privilege-settings.sql

```diff
@@ -0,0 +1,13 @@
+SET NAMES utf8mb4;
+START TRANSACTION;
+
+INSERT INTO changelog VALUES (28, UNIX_TIMESTAMP());
+
+CREATE TABLE privileges (
```

Comment:
Every table _MUST_ have a primary key. In this case it should be the `privilege` field